modules/serverless-rn-1-27-0.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-27_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.27
+
+{ServerlessProductName} 1.27 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1-27_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.6.
+* {ServerlessProductName} now uses Knative Eventing 1.6.
+* {ServerlessProductName} now uses Kourier 1.6.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.6.
+* {ServerlessProductName} now uses Knative Kafka 1.6.
+* The `kn func` CLI plug-in now uses `func` 1.8.1.
+
+* {ServerlessProductName} 1.27 is the earliest release that is fully supported on {product-title} 4.12. {ServerlessProductName} 1.25 and older does not deploy on {product-title} 4.12.
++
+Consequently, before upgrading {product-title} to version 4.12, first upgrade {ServerlessProductName} to version 1.26 or newer.
+
+* Namespace-scoped brokers are now TP. Such brokers can be used, for instance, to implement role-based access (RBAC) policies.
+
+* `KafkaSink` now uses the `CloudEvent` binary content mode by default. The binary content mode is more efficient than the structured mode because it uses headers in its body instead of a `CloudEvent`. For example, for the HTTP protocol, it uses HTTP headers.
+
+* You can now use the gRPC framework over the HTTP/2 protocol for external traffic using the OpenShift Route on {product-title} 4.10 and later. This improves efficiency and speed of the communications between the client and server.
+
+* API version `v1alpha1` of the Knative Operator Serving and Eventings CRDs is deprecated in 1.27. It will be removed in future versions. Users are strongly recommended to use the `v1beta1` version instead. This does not affect the existing installations, because CRDs are updated automatically when upgrading the Serverless Operator.
```

Comment:
the knativeserving and knativeeventing v1alpha1 deprecation should also appear in the "Table 2. Deprecated and removed features tracker "